docs/mkdocs.yml

```diff
@@ -0,0 +1,143 @@
+theme:
+  name: material
+  logo: assets/logo.png
+  features:
```

Comment:
are there any of the features that are debatable (i.e. that it would be worth to decide if we turn them on/off - there are many of them and for most I do not understand in 100% what they do 😄)?

Reply:
I only use the ones that make sense to me 😄 . And I left the others just in case someone wants to play with these settings.